backend/src/Squidex.Domain.Apps.Entities/Contents/Queries/ContentQueryService.cs

```diff
@@ -92,6 +92,11 @@ public async Task<IResultList<IEnrichedContentEntity>> QueryAsync(Context contex
 
             var schema = await GetSchemaOrThrowAsync(context, schemaIdOrName);
 
+            if (context.Permissions.Any(p => p.Id == Permissions.AppContentReadOwn))
```

Comment:
Use could accidentally have other permissions that allow access to the content. Furthermore the permissions are not concrete, they contain placeholders. You have to use Permission.ForApp(<Permission>, appName, schemaName);

Reply:
what should I pass in param **Id**?

Reply:
Then permission